### ISSUE TITLE
## 3일차

### ISSUE THREAD

**Issue by socome** [sejong-rcv/2019.KCCV#3]
1. Strand-accurate Multi-view Hair Capture (KAIST,김민혁)

2. SFNet: Learning Object-aware Semantic Correspondence (Yonsei, 함범섭)

3. PoseFix: Model-agnostic General Human Pose Refinement Network (SNU, 이경무)

4. Efficient Relative Attribute Learning using Graph Neural Networks (KU, 김현우)

5. Deep Metric Learning Beyond Binary Supervision (POSTECH,곽수하)

6. Learning Loss for Active Learning (KAIST,권인소)

7.  Joint manifold diffusion for combining predictions on decoupled observations (UNIST,김광인)

8. Attention-based Dropout Layer for Weakly Supervised Object Localization (Yonsei, 심현정)

9. Semantic Attribute Transfer Matching (École polytechnique fédérale de Lausanne, 김승룡)

**Comment by Namhyunho** [comment]
Strand-accurate Multi-view Hair Capture (KAIST,김민혁)

SFNet: Learning Object-aware Semantic Correspondence (Yonsei, 함범섭)

**Comment by socome** [comment]
SFNet: Learning Object-aware Semantic Correspondence

Semantic Correspondence

서로다른 고양이 영상이 있을 때, 고양이 눈,귀,코 와 같이 Semantic 한 정보를 
연결짓는 것을 의미한다. 여기서 가장 기본적인 전제는 같은 클래스의 이미지라고
생각한다는 것이다.  Semantic Correspondence은 stereo matching와 비슷하지만 좀 더 General 하고 challenging한 Task와 연관있다. 

Semantic Correspondence의 응용분야로는 Single view 3D reconstruction, Robotics, Colorization,
Cosegmentation 등이 있다.
 
Semantic Correspondence의 Previous work로는 SIFT, Defomable Spatial pyramid, Proposal Flow와 같이 Hand-crafted feature를 통한 방법들도 존재했고, CNN이 등장하면서 딥러닝화하려고 연구되고 있고, SCNet이나 FCSS 등이 등장했다.

딥러닝으로 Semantic Correspondence를 하기위해서 필요한 데이터(GT)는 수집하기 어렵다. 이를 해결하기 위해서 CNNGeo라는 방법들이 나오는 등 꾸준히 Semantic Correspondence를 위한 네트워크들이 발표되고 있다. 

Semantic Correspondence의 한계는 다음과 같다.
Hand-crafted features
- Lack of high level semantics
- Susceptible to distractions
Lack of differentiability
- Training descriptor only
- Incorporating hand-designed matching algorithms
Lack of flexibility
- Global transformation cannot handle non-rigid deformations
추가로 데이터 셋이 절대적으로 부족하다는 한계가 있다. 

해당 논문에서의 Motivation은
object mask,
